templates/taxi/car_detail.html

```diff
@@ -12,8 +12,23 @@ <h1>
     </a>
   </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
+  <div style="display: flex; justify-content: space-between; align-items: center;">
+    <h1>Drivers</h1>
+    <div>
+        {% if request.user in object.drivers.all %}
```

Comment:
The condition `request.user in object.drivers.all` should be `request.user in car.drivers.all` to correctly reference the car object being displayed. Ensure that the variable name matches the context provided by the view.